package.json

```diff
@@ -38,6 +38,7 @@
     "firebase": "^8.2.3",
     "geolib": "^3.3.3",
     "i18n-js": "^4.1.1",
+    "moment": "^2.29.4",
```

Comment:
Moment has been depreceted

Reply:
I was using locale String, but I could not get the exact format used for the date in the mockup. So I have used moment for the date for conversion.